synapse/module_api/__init__.py

```diff
@@ -768,6 +801,11 @@ def generate_short_term_login_token(
                to get this token, if any. This is encoded in the token so that
                /login can report stats on number of successful logins by IdP.
         """
+        warnings.warn(
```

Comment:
Does Synapse actually show these? I've quickly tried printing one on a local instance and failed.

Reply:
I also failed to do that. I changed that to a `logger.warn` call instead